packages/react/src/components/ColorPicker/ColorSlider/ColorSlider.base.tsx

```diff
@@ -196,12 +208,44 @@ export class ColorSliderBase extends React.Component<IColorSliderProps, IColorSl
     this._updateValue(ev, newValue);
   };
 
+  private _onTouchStart = (ev: TouchEvent): void => {
+    if (!this._root.current) {
+      return;
+    }
+
+    // prevent touch from scrolling the page so that the touch can be dragged on the color rectangle.
+    ev.stopPropagation();
+  };
+
+  private _onTouchMove = (ev: TouchEvent): void => {
+    if (!this._root.current) {
+      return;
+    }
+
+    const lastTouch = ev.touches[ev.touches.length - 1];
+    if (lastTouch.clientX !== undefined && lastTouch.clientY !== undefined) {
+      const maxValue = this._maxValue;
+      const rectSize = this._root.current.getBoundingClientRect();
```

Comment:
Calling `getBoundingClientRect()` here is potentially a performance issue as it will trigger layout. Will `rectSize` change will touch is moving? Could this value be computed on touch start and cached for move?